src/arithmetic/fields.rs

```diff
@@ -130,12 +107,6 @@ pub trait FieldExt: ff::PrimeField + From<bool> + Ord + Group<Scalar = Self> {
     /// Gets the lower 32 bits of this field element when expressed
     /// canonically.
     fn get_lower_32(&self) -> u32;
```

Comment:
Don't we use `get_lower_32` in the implementation of `sqrt_ratio`?

(Technically we only need a hash that is distinct for all elements of the 2<sup>S</sup> order subgroup; not necessarily the low 32 bits.)

Reply:
We were also using it in `map_to_curve_simple_swu` to implement `sgn0`, but I've changed that to use `PrimeField::is_odd`
.